sos/plugins/origin.py

```diff
@@ -119,6 +119,9 @@ def setup(self):
             # Note: Information about nodes, events, pods, and services
             # is already collected by the Kubernetes plugin
             self.add_cmd_output([
+                "openshift vesion",
+                "%s vesion" % self.oc_cmd,
+                "%s vesion" % self.oadm_cmd,
```

Comment:
Yes, however the version information collected by this is for user convenience (to the user looking at the data), I will remove some of the duplication.